src/routes/Rockets.js

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import getRockets from '../redux/rockets/actions';
+import Rocket from '../components/Rocket';
+
+const Rockets = () => {
+  const { rockets, isLoading, error } = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getRockets());
+  }, [dispatch]);
```

Comment:
- [ ] You did a good job dispatching the fetch actions in their respective components. :+1:  But there seems to be a small issue here, let's say I reserved some rockets and visited the profile page. If I go back to the Rockets page, everything will be reset. This is happening because of the useEffect here making an API call every time I'm navigating to the page. You can prevent this issue by utilizing a guard. 🧐️

**Example**
```
useEffect(() => {
  if (rockets.length===0) {
    dispatch(getRockets());
  }
}, [dispatch]);
```

**Project requirements**
![image](https://user-images.githubusercontent.com/17243325/232020027-23203fa0-954d-43a5-82ea-e57c047c65f4.png)